src/main/resources/reference.conf

```diff
@@ -751,16 +787,19 @@ resourceTypes = {
     ownerRoleName = "steward"
     roles = {
       steward = {
-        roleActions = ["delete", "edit_datasnapshot",  "read_data",  "discover_data",  "share_policy::steward", "share_policy::custodian", "share_policy::reader",  "share_policy::discoverer", "read_policies", "alter_policies"]
+        roleActions = ["delete", "edit_datasnapshot", "update_snapshot", "read_data",  "discover_data",  "share_policy::steward", "share_policy::custodian", "share_policy::reader",  "share_policy::discoverer", "read_policies"]
       }
       custodian = {
-        roleActions = ["delete", "edit_datasnapshot",  "read_data",  "discover_data",  "share_policy::reader",  "share_policy::discoverer", "read_policies"]
+        roleActions = []
       }
       discoverer = {
-        roleActions = ["discover_data", "read_policy::custodian"]
+        roleActions = ["discover_data", "read_policy::steward", "read_policy::discoverer"]
       }
       reader = {
-        roleActions = ["read_data", "discover_data", "read_policy::custodian"]
+        roleActions = ["read_data", "discover_data", "read_policy::steward", "read_policy::discoverer"]
```

Comment:
Is this a breaking change?

Reply:
I went back and forth on this.  Ultimately, no, since we're getting rid of custodian.  That being said, I'll re-add until I change the UI to not request the custodians